variables.tf

```diff
@@ -246,10 +274,10 @@ variable "target_groups" {
   }))
 }
 
-variable "dns_zone_id" {
-  description = "Zone ID of the hosted zone"
-  type        = string
-}
+#variable "dns_zone_id" {
+#  description = "(Required) Zone ID of the hosted zone where records for LB will be created"
+#  type        = string
+#}
```

Comment:
Can probably just be removed rather than commented.

Reply:
done